ONSdigital/ras-frontstage
# update pipenv

# What and why?
Removes security vulnerabilities by updating pipenv packages. Also introduces two ignore codes for djlint which preventing linting errors introduced after it was updated which will need to be addressed in another card. 
# How to test?
Run the tests and check the add survey function works correctly as the redirect had to be updated.
# Jira
https://jira.ons.gov.uk/secure/RapidBoard.jspa?rapidView=1493&projectKey=RAS&view=detail&selectedIssue=RAS-783
